Index: Estructura  y Restricciones Declarativas/Pruebas/PoblarOK.sql
===================================================================
diff --git a/Estructura  y Restricciones Declarativas/Pruebas/PoblarOK.sql b/Estructura  y Restricciones Declarativas/Pruebas/PoblarOK.sql
new file mode 100644
--- /dev/null	(date 1743191886855)
+++ b/Estructura  y Restricciones Declarativas/Pruebas/PoblarOK.sql	(date 1743191886855)
@@ -0,0 +1,5 @@
+INSERT INTO CENTROSDEESTUDIOS (idCentroEstudios, nombre, idDirector, idProgramaAcademico)
+VALUES ('CEIS', 'Centro de Estudios en Ingeniería de Software', '', '');
+
+INSERT INTO MATERIAS (idMateria, nombre, creditos, horasTeoricas, horasPracticas, nivel, modalidad, idArea, idCentroDeEstudios, idNucleoDeFormacion)
+VALUES ('MBDA', 'Modelos y Bases de Datos',4 ,1.5 , 1.5, '', 'Presencial', '', '', '');
\ No newline at end of file
diff --git a/Estructura  y Restricciones Declarativas/Persistencia/Consultas.sql b/Estructura  y Restricciones Declarativas/Persistencia/Consultas.sql
new file mode 100644
diff --git a/Estructura  y Restricciones Declarativas/Pruebas/PoblarNoOK.sql b/Estructura  y Restricciones Declarativas/Pruebas/PoblarNoOK.sql
new file mode 100644
diff --git a/Estructura  y Restricciones Declarativas/Pruebas/XPoblar.sql b/Estructura  y Restricciones Declarativas/Pruebas/XPoblar.sql
new file mode 100644
